Model/Queue.php

```diff
@@ -64,32 +82,35 @@ public function __construct(
         $this->maxSingleJobDataSize = $this->configHelper->getNumberOfElementByPage();
     }
 
-    public function addToQueue($className, $method, $data, $data_size = 1)
+    /**
+     * @param $className
+     * @param $method
+     * @param $data
+     * @param int $dataSize
+     * @param bool $isFullReindex
+     */
+    public function addToQueue($className, $method, $data, $dataSize = 1, $isFullReindex = false)
     {
         if (is_object($className)) {
             $className = get_class($className);
         }
 
         if ($this->configHelper->isQueueActive()) {
-            $this->insert($className, $method, $data, $data_size);
+            $this->db->insert($this->table, [
+                'created'   => date('Y-m-d H:i:s'),
+                'class'     => $className,
+                'method'    => $method,
+                'data'      => json_encode($data),
+                'data_size' => $dataSize,
+                'pid'       => null,
+                'is_full_reindex' => $isFullReindex ? 1 : 0,
```

Comment:
Is it a column that have already been added before ? I don't have it so I get an error on indexing

Reply:
/o\ Great catch, @damcou!! Thank you!
I added it manually to my DB and completely forgot to add it to `Upgrade` scripts of the extension /o\

Reply:
@damcou Fixed. You can go to `setup_module` table, lower version of Algolia module to `1.9.0` and run `php bin/magento setup:upgrade` command. It'll create the column :)

Reply:
Thanks :)